src/hotspot/os/aix/os_aix.cpp

```diff
@@ -2998,3 +2999,11 @@ bool os::supports_map_sync() {
 }
 
 void os::print_memory_mappings(char* addr, size_t bytes, outputStream* st) {}
+
+void os::print_user_info(outputStream* st) {
+  os::Posix::print_user_info(st);
+}
+
+void os::print_active_locale(outputStream* st) {
+  os::Posix::print_active_locale(st);
+}
```

Comment:
You should not need this indirection. The code in os::Posix::x should be the implementation os::x for the os class, but just defined in os_posix.cpp.